mypyc/test-data/irbuild-bytes.test

```diff
@@ -185,3 +185,15 @@ L0:
     r10 = CPyBytes_Build(2, var, r9)
     b4 = r10
     return 1
+
+[case testDecodeUtf8]
+def f(b: bytes) -> str:
+    return b.decode("utf-8")
```

Comment:
Well it's silly to have a parameter when we only support `"utf-8"`. It doesn't look like you have a fallback either but I don't really know mypyc.